pw-pbkdf2.c

```diff
@@ -229,6 +246,95 @@ static int pbkdf2_encrypt(
 						  dk.bv_len, iteration,
 						  salt.bv_len, (const uint8_t *) salt.bv_val,
 						  dk.bv_len, dk_value);
+#elif HAVE_MOZNSS
+	SECStatus rv;
+	rv = NSS_NoDB_Init(".");
+	if (rv != SECSuccess)
+ 	{
+    		fprintf(stderr, "NSS initialization failed (err %d)\n",
+        	PR_GetError());
+  	}
+        PK11SlotInfo *slot = NULL;
```

Comment:
Some Indentations are wrong, tab indentation are used in upstream OpenLDAP project.

Reply:
ok i retabulated the whole file